connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedConnectCluster.java

```diff
@@ -263,36 +266,37 @@ public void startConnect() {
      *
      * @param connName   the name of the connector
      * @param connConfig the intended configuration
-     * @throws IOException          if call to the REST api fails.
+     * @throws ConnectException if the configuration fails to be serialized
      * @throws ConnectRestException if REST api returns error status
```

Comment:
Don't we also need to change the description to something like:
```suggestion
     * @throws ConnectRestException if REST api returns error status or the REST API fails
```